assets/javascripts/test_result.js

```diff
@@ -586,6 +586,21 @@ function setCurrentPreviewFromStepLinkIfPossible(stepLink) {
   }
 }
 
+function githash_to_link(value) {
```

Comment:
That's not Perl, so please use JavaScript naming conventions, e.g.
```suggestion
function githashToLink(value) {
```
But also change the other occurrences.

Reply:
Done